dispatch/BESContainerStorageList.cc

```diff
@@ -44,12 +45,13 @@
 
 #include "BESDebug.h"
 
-
+using namespace std;
```

Comment:
This doesn't look like it's needed. This might be true elsewhere - It looks like you always qualify the mutex-related calls with 'std::'. Look for this in other files and remove it if possible.